kubecluster.yaml

```diff
@@ -77,12 +85,12 @@ parameters:
   docker_registry_url:
     type: string
     description: Docker registry url
-    default:
+    default: ""
```

Comment:
Exactly my thoughts thought too. Including the double quotes however seemed to fix the problem.